File: Java/src/main/java/QuestionBank/Solution001_100/Solution58.java
package QuestionBank.Solution001_100;

/**
 * @author windows
 */
public class Solution58 {
    public int lengthOfLastWord(String s) {
        String[] str = s.split(" ");
        return str[str.length - 1].trim().length();
    }


    public static void main(String[] args) {
        Solution58 solution58 = new Solution58();
    }
}

File: C++/src/Solution1_100/Solution58.cpp
//
// Created by windows on 2023/3/2.
//

#include "Solution58.h"
#include <iostream>
#include <vector>
#include <unordered_set>
#include <unordered_map>
#include <string>
using namespace std;

int Solution58::lengthOfLastWord(string s) {
    int len = 0;
    int i = s.length() - 1;
    while (i >= 0 && s[i] == ' '){
        i--;
    }
    while (i >= 0 && s[i] != ' '){
        i--;
        len++;
    }
    return len;
}
